packages/jest-reporters/src/utils.ts

```diff
@@ -99,6 +138,10 @@ export const getSummary = (
     runTime = Math.floor(runTime);
   }
 
+  const valuesForCurrentTestCases = getValuesCurrentTestCases(
+    options ? options.currentTestCases : [],
```

Comment:
```suggestion
    options?.currentTestCases,
```

it's defaulted to empty array in the function, no need to do that here